packages/stryker-webpack-transpiler/src/compiler/ConfigLoader.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import { Configuration } from 'webpack';
+import { StrykerWebpackConfig } from '../WebpackTranspiler';
+import { getLogger } from 'log4js';
+
+const PROGRESS_PLUGIN_NAME = 'ProgressPlugin';
+
+export default class ConfigLoader {
+  private log = getLogger(ConfigLoader.name);
+
+  public constructor(private loader: NodeRequireFunction = require) {
```

Comment:
My personal preference is to explicitly define the private variables instead of using the TypeScript shortcut, I think this makes the code clearer for people that haven't worked with TypeScript.

Reply:
I understand that sentiment. However, that would mean having to repeat myself (variable name + type). The benefits out way the readability argument here.

It took me a while to get use to this syntax as well, but in the end: people should know the language (at least a little bit) before contributing.